docker/mssql-techempower/import-data.sh

```diff
@@ -0,0 +1,5 @@
+#wait for the SQL Server to come up
+sleep 90s
```

Comment:
I would try to find a way to avoid this 90s delay since it makes building the container quite slow, but it's OK for now.

Reply:
This is from the image documentation, we can go smaller but this is what they use.